packages/edit-post/src/components/layout/style.scss

```diff
@@ -59,6 +59,7 @@
 
 .interface-interface-skeleton__sidebar > div {
 	height: 100%;
+	padding-bottom: $grid-unit-60;
```

Comment:
Adds some extra padding at the bottom. Panels near won't feel so "tight"

Reply:
Is  this specific to the edit-post sidebar or all interface sidebars.
I know this selector was already here but it seems like it's too generic  while added in `edit-post`?

Reply:
I suppose all Interface sidebars. I added it here for now, as I wasn't familiar with the Interface components, and wanted to limit scope.